src/services/highcharts-theme/highcharts-theme.svc.coffee

```diff
@@ -0,0 +1,32 @@
+angular
+.module('impac.services.highcharts-theme', [])
+.service('HighchartsThemeService', (ImpacTheming) ->
```

Comment:
Looks more like something that should live in an angular `.run()` function?

Reply:
I am having issues with `Highcharts.theme =` where Highcharts is not defined yet. Not being a service (i.e. no HighcartsProvider), I am not sure this can live in a run() function